presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -286,6 +286,8 @@ Name                                        Description
 ``running-mode``                            Set to ``query-bank`` to make the Verifier run in ``query-bank`` mode. Supports
                                             ``query-bank`` and ``control-test``. Defaults to ``control-test``.
 ``save-snapshot``                           Set to ``true`` to save checksums to ``mysql`` database.
+``extended-verification``                   Set to ``true`` to run extended verification. It only applies to ``Insert``and ``CreateTableAsSelect`` queries.
```

Comment:
```suggestion
``extended-verification``                   Set to ``true`` to run extended verification. It only applies to ``Insert`` and ``CreateTableAsSelect`` queries.
```
Nit: missing space causes formatting error in local build of docs. 
<img width="954" alt="Screenshot 2024-02-27 at 10 18 54 AM" src="https://github.com/prestodb/presto/assets/7013443/18eb5e11-dac1-4928-99d8-4fb2819850c0">